config/terraform/application/config/production.tfvars.json

```
@@ -0,0 +1,9 @@
+{
+    "cluster": "production",
+    "namespace": "cpd-production",
+    "environment": "production",
+    "enable_postgres_backup_storage" : true,
+    "enable_monitoring": true,
+    "external_url": "https://ec2.production.teacherservices.cloud/healthcheck",
+    "statuscake_contact_groups": [282453]
```

I think this should be the CPD contact group.

```suggestion
    "statuscake_contact_groups": [291418]
```